static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -66,9 +68,11 @@ const setDfpListeners = (): void => {
 };
 
 const setPersonalisedAds = (): void => {
+    const cookieName = 'GU_PERSONALISED_ADS_GOOGLE';
     if (config.switches.includePersonalisedAdsConsent) {
-        // TODO: replace this hardcoded value with one from local storage
-        const personalised = false;
```

Comment:
With #19649 you'll be able to use a nicer `getAdConsentState(thirdPartyTrackingAdConsent)` that returns a normal boolean and abstracts away the cookie reading!

Reply:
Sounds much better.